util/deploy-to-maven-central.sh

```diff
@@ -60,4 +50,5 @@ for generated_pom_file in dagger*pom.xml; do
   rm "${generated_pom_file}.asc"
 done
 
-
+git tag dagger-"${version_name}"
```

Comment:
You may want to use annotated tags instead (works better with git describe et al)

Reply:
So `git tag -a -m "Dagger ${version_name}" dagger-${version_name}"`?

I guess it's good to have the extra info on when the tag is actually made.

Reply:
That's what I generally use yes.